src/AlpineSkiHouse.Web/Events/PassPurchased.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace AlpineSkiHouse.Events
+{
+    public class PassPurchased
+    {
+        public int PassTypeId { get; set; }
+        public int CardId { get; set; }
+
+        public decimal PricePaid { get; set; }
+
+        public string DiscountCode { get; set; }
```

Comment:
I always like to see dates on events but it is less important for internal messages.